### ISSUE TITLE
Downloads stopped working?

### ISSUE THREAD

**Issue by trvswgnr** [bogem/nehm#20]

**Comment by n10v** [comment]
Yes, suddenly the rate limit is over: https://api.soundcloud.com/tracks/256347798/stream?client_id=11a37feb6ccc034d5975f3f803928a32

**Comment by bogem** [comment]
I will try to fix it

**Comment by n10v** [comment]
Soundcloud provides only 15k downloads in 24-hour window: https://developers.soundcloud.com/docs/api/rate-limits#play-requests, so I can't handle that and expand rate limits. The only thing I can do is to add the appropriate error message.

**Comment by n10v** [comment]
Ideal solution: register more apps and if rate limit is exceeded on one client_id then use another client_id.
But SoundCloud doesn't allow now to register new apps.
